docs/user/reference/metrics/event.md

```diff
@@ -187,6 +190,10 @@ var snapshot = views::login_opened.test_get_value(None).unwrap();
 assert_eq!(2, snapshot.len());
 let first = &snapshot[0];
 assert_eq!("login_opened", first.name);
+
+let extra = event.extra.unwrap();
+assert_eq!(1, extra.len());
+assert_eq!("toolbar", extra["source_of_login"]);
```

Comment:
Does this syntax work? I don't think Rust has subscript operator overloading like this and we provide concrete structs, so this should maybe be `extra.source_of_login`?